lms/static/js/dialog_tab_controls.js

```diff
@@ -0,0 +1,77 @@
+var DialogTabControls = (function() {
+    'use strict';
+
+    var focusableChildren,
+        numElements,
+        currentIndex,
+        setCurrentIndex = function(currentElement) {
```

Comment:
@cahrens Also added this new function to set correct current elements index to tab correctly if someone click on any input field out of tab order.